src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = (props) => {
```

Comment:
Use destructuring for the props
```suggestion
export const Person = ({ person }) => {
```